library/classes/Document.class.php

```diff
@@ -762,4 +762,13 @@ function postPatientNote($provider, $category_id, $message = '')
         // Link the new note to the document.
         setGpRelation(1, $this->get_id(), 6, $noteid);
     }
+      
+  /**
+   * Return note objects associated with this document using Note::notes_factory
+   *
+   */
+    function get_notes()
+    {
+        return (Note::notes_factory($this->get_id()));
```

Comment:
Since notes_factory is not a static function, don't you need to create a new Note() instance first here?